rest/utilities_testing_resttester.go

```diff
@@ -131,7 +131,11 @@ func (rt *RestTester) WaitForVersion(docID string, version DocVersion) error {
 		}
 		var body db.Body
 		require.NoError(rt.TB(), base.JSONUnmarshal(rawResponse.Body.Bytes(), &body))
-		return body.ExtractRev() == version.RevID
+		if body.ExtractRev() != version.RevID {
+			rt.TB().Logf("Retrying WaitForVersion for doc %s, expected rev %s, got body %s", docID, version.RevID, body)
```

Comment:
This is probably going to end up being quite noisy for test, if you want to change this I'd probably use require.EvenutallyWithT and have the log messages on the assertion, so that you see the error on the last failing pass.